source/agora/test/Base.d

```diff
@@ -258,9 +261,7 @@ public NetworkT makeTestNetwork (NetworkT : TestNetworkManager)
             configs ~= conf;
         }
 
-
-        auto net = new NetworkT(NodeConfig.init, node_configs.map!(
-            c => c.key_pair.address.toString()).array);
```

Comment:
I think that's where our approaches differ. I think that those arguments should be used / useful, and I thought they currently were, at least for the second one ?

Reply:
Well it makes sense in the context of your previous PR which only uses `apis`. But if we go this way, we do not need, nor do we want, to have a `NetworkManager`, because we're essentially conflating 2 use cases into one.

Reply:
Yeah that's why I suggested splitting up these into two different classes. But if keep them the same class, the thing is these arguments that we pass here are just not used at all in the test-suite. Which approach do we take?